scalding-base/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -951,11 +942,6 @@ object Execution {
   /** Returns a constant Execution[Unit] */
   val unit: Execution[Unit] = from(())
 
-  /**
-   * This converts a function into an Execution monad. The flowDef returned is never mutated.
-   */
-  def fromFn(fn: (Config, Mode) => FlowDef): Execution[Unit] =
```

Comment:
this is not easy to add back, and I'd rather not. It's exposing too much of cascading even in cascading mode.

@nownikhil can you check to see if `Execution.fromFn` is used at all at Twitter? There are only a couple calls to it in this repo for some tests, but I think it's a bad idea.

Reply:
I see a few uses of it at Twitter, which I suspect are legacy jobs. But more importantly, it's used in Summingbird [here](https://github.com/twitter/summingbird/blob/develop/summingbird-scalding/src/main/scala/com/twitter/summingbird/scalding/ScaldingPlatform.scala#L610).